test/e2e/internal/testdata/foobar/const.go

```diff
@@ -16,15 +16,19 @@ limitations under the License.
 package foobar
 
 import (
+	"path/filepath"
 	"github.com/opencontainers/go-digest"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"oras.land/oras/test/e2e/internal/utils/match"
 )
 
 var (
-	Tag              = "foobar"
-	Digest           = "sha256:fd6ed2f36b5465244d5dc86cb4e7df0ab8a9d24adc57825099f522fe009a22bb"
-	ManifestStateKey = match.StateKey{Digest: "fd6ed2f36b54", Name: "application/vnd.oci.image.manifest.v1+json"}
+	Tag                     = "foobar"
+	Digest                  = "sha256:fd6ed2f36b5465244d5dc86cb4e7df0ab8a9d24adc57825099f522fe009a22bb"
+	ManifestStateKey        = match.StateKey{Digest: "fd6ed2f36b54", Name: "application/vnd.oci.image.manifest.v1+json"}
+	PermissionDirectoryName = "test-permission-dir"
+	PermissionFileName      = filepath.Join(PermissionDirectoryName, "test-permission-file")
```

Comment:
For future maintainability, can we add small comment to mention the permissions of the file? Something like:

```suggestion
	PermissionFileName      = filepath.Join(PermissionDirectoryName, "test-permission-file") // permissions: 777
```